### PR TITLE
Add sprint report for Sprint 26

### PR DESCRIPTION
**Pull request by DavidKang**

### CODE REVIEW

File: _posts/development/2017-10-30-sprint-report-11.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+### Kiwi Editor - minor workflow change
+
+We made a minor workflow change, now after an image template with a kiwi file is branched the user will be redirected to the Image Editor instead of Package show.
+
+## Bugfixes
+
+### Kiwi Editor - Breadcrumb and flash messages
+
+We added the missed link for the package that contains the Kiwi image description is in the breadcrumb and we also improved the flash error messages to be more understandable in the Kiwi Editor. At the beginning the messages were shown as `Repository[0] alias ...`. To be honest, this is not a great help to identify which element fails in a hunge bunche of repository and packages. And the result of this improvement looks like:
```
Comment (evanrolfe) [inline _posts/development/2017-10-30-sprint-report-11.md]: > We added the missed link for the package that contains the Kiwi image description is in the breadcrumb

This is kind of hard to read,  maybe say this instead? `We added a missing breadcrumb link for the packages that contain a Kiwi image description`

> which element fails in a hunge bunche of repository and packages

Hunge => huge
bunche => bunch
Though maybe better to say:
`this is not a great help to identify which elements are invalid when you have a large amount of repositories and packages.`

File: _posts/development/2017-10-30-sprint-report-11.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+![](https://user-images.githubusercontent.com/1212806/31459380-4e9f61b4-aec3-11e7-9058-6457b4792ea2.png)
+
+We encourage you to take a look at the PR [#3981](https://github.com/openSUSE/open-build-service/pull/3981) to have more details.
+
+### Issue #3897 - Request does not include <history>element for <priority>change</priority></history>
+
+The [issue](https://github.com/openSUSE/open-build-service/issues/3897) was reported by [jberry-suse](https://github.com/jberry-suse) some days ago and we spent this sprint to investigate what is causing this. Read on, if you want to know what was going on.
+
+When the OBS saves a request it runs a sanitization job to ensure that every stored request is in a valid form. Part of this sanitization is to check whether there are any associated request actions that have a higher priority than the request itself. If that's the case, OBS raises the priority of the request.
```
Comment (evanrolfe) [inline _posts/development/2017-10-30-sprint-report-11.md]: > When the OBS saves a request

Should be `When OBS saves a request`

**Comment by bgeuken** [comment]
Looks good. It seems like some things we worked on are missing

File: _posts/development/2017-10-30-sprint-report-11.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+This automatic priority change was not logged via a history element, unlike we do for changes made by users. So the cause of this priority change was invisible to users and thus caused confusion. To solve this we now track automatic priority changes via a history element as well.
+
+
+## Test suite migration
+
+This time we only write rspec tests for app/models/attrib_value.rb, [PR#3950](https://github.com/openSUSE/open-build-service/pull/3950). But for sure next sprint we will have more ^^.
```
Comment (evanrolfe) [inline _posts/development/2017-10-30-sprint-report-11.md]: Should be `only wrote rspec tests`

**Comment by DavidKang** [comment]
@bgeuken, @evanrolfe and @eduardoj could you review it again, please 😸. I added missing content.

**Comment by bgeuken** [comment]
@DavidKang Well done! Thanks for taking care of this:heart:

**Pull request merged**